Cargo.toml

```diff
@@ -1,17 +1,13 @@
 [package]
 name = "xmtp-message-history-server"
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
Do we use these versions anywhere?

Reply:
i don't think so, but i bumped the version just in case since there are breaking changes